unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/EntityInformation/EntityInformationController.cs

```diff
@@ -125,7 +125,7 @@ public void SetEntity(DCLBuilderInWorldEntity entity, ParcelScene currentScene)
         if (entity.HasSmartItemComponent())
         {
             if(entity.rootEntity.TryGetBaseComponent(CLASS_ID_COMPONENT.SMART_ITEM, out BaseComponent baseComponent))
-                smartItemListView.SetSmartItemParameters(entity.GetSmartItemParameters(), ((SmartItemComponent) baseComponent).model.values);
+                smartItemListView.SetSmartItemParameters(entity.GetSmartItemParameters(), ((SmartItemComponent) baseComponent).GetValues());
```

Comment:
Reminder for Santi: It will provoke conflicts on your current branch -> `chore/BuildModeHUD-refactorization`